src/View/mainpage/MainPage.py

```diff
@@ -94,6 +96,23 @@ def setup_ui(self, main_window_instance):
         self.dicom_tree = DicomTreeView()
         self.right_panel.addTab(self.dicom_tree, "DICOM Tree")
 
+        # Create Clinical Data tab
+        # TODO refactor the entire Clinical Data form/display class
+        # As they currently stand, they are given the right tab widget, and make direct modifications to the tab.
+        # This class should be refactored in the same way as the rest of the main window's components, i.e. the Clinical
+        # Data should be a child of QWidget that can exist independently of OnkoDICOM. This class would differ from most
+        # other main window components in that rather than interacting with the PatientDictContainer as its model, it
+        # would use the patient's ClinicalData csv file as the model (which means that the QWidget would theoretically
+        # easily exist outside OnkoDICOM).
+        # There are two classes: one for displaying the clinical data, and another for modifying the clinical data.
+        # The check below determines whether there already exists a clinical data csv for the patient, and loads either
+        # the data display or the data form depending on what exists.
```

Comment:
"Vision statement" for the refactor of the Clinical Data tab. As it stands, the component isn't bad, it's just inconsistent with the other main window components.